launcher/DashFactionLauncher.rc

```diff
@@ -126,13 +126,14 @@ BEGIN
     AUTOCHECKBOX    "Fast animations",IDC_FAST_ANIMS_CHECK,192,150,78,10
     AUTOCHECKBOX    "High model details",IDC_DISABLE_LOD_CHECK,192,162,78,10
 
-    GROUPBOX        "Miscellaneous",IDC_STATIC,6,192,270,84
+    GROUPBOX        "Miscellaneous",IDC_STATIC,6,192,270,100
     AUTOCHECKBOX    "Fast Start",IDC_FAST_START_CHECK,12,204,66,10
     AUTOCHECKBOX    "Allow overriding game files by packages in user_maps",IDC_ALLOW_OVERWRITE_GAME_CHECK,12,216,260,10
     AUTOCHECKBOX    "Run game at reduced speed when window doesn't have focus",IDC_REDUCED_SPEED_IN_BG_CHECK,12,228,260,10
+    AUTOCHECKBOX    "Derandomize Bomb", IDC_DERANDOMIZE_BOMB, 12,240,80,10
```

Comment:
I believe we don't need a setting for that, especially that it is a spoiler

Reply:
I don't know how much I like implementing this without a setting - it would be a fundamental change to the logic of the end of the game in a way that may reasonably not be desired by all players. It's not an upgrade necessarily, it's just an alternate approach. I personally think it's a very positive change, but still would like it to be a setting - though, I wouldn't necessarily be opposed to it defaulting to on.

I don't necessarily think it being a spoiler is a huge issue if it's phrased this way with no tooltip or helper text. "Derandomize bomb" doesn't really mean much without context or knowledge of the end of the game. Though, maybe "Improve endgame sequence" or something more cryptic should be considered if that were a concern.